bigquery/samples/tests/conftest.py

```diff
@@ -25,6 +25,22 @@ def client():
     return bigquery.Client()
 
 
+@pytest.fixture
+def project_id(client):
+    return client.project
+
+
+@pytest.fixture
+def table_id(client, dataset_id):
+    now = datetime.datetime.now()
+    table_id = "example_table_{}_{}".format(
+        now.strftime("%Y%m%d%H%M%S"), uuid.uuid4().hex[:8]
+    )
+    table = client.create_table("{}.{}".format(dataset_id, table_id))
```

Comment:
Since you're creating a table in your sample, you don't actually want to create one in the test fixture.

We might want to rename this to `random_table_id` to indicate that no table is actually created.

Reply:
I renamed the fixture to `random_table_id` and removed the creation of the table in the fixture.